source/_integrations/ohme.markdown

```diff
@@ -143,6 +143,47 @@ The `ohme.set_price_cap` action is used to set the price cap threshold. This can
 | `config_entry`         | No       | The config entry of the account to apply the price cap to.  |
 | `price_cap`            | No       | Threshold in 1/100ths of your local currency.               |
 
+
+## Use cases
+
+This integration can be used to show charger status on your dashboards. It can also be used as part of automations for solar and battery storage.
+
+
+## Examples
+
+### Send a notification when a vehicle is plugged in:
+{% raw %}
+```yaml
+# Example automation
+mode: single
+triggers:
+  - trigger: state
+    entity_id:
+      - sensor.ohme_home_pro_status
+    from: unplugged
+conditions: []
+actions:
+  - action: notify.mobile_app_iphone
+    data:
+      message: Vehicle plugged in
```

Comment:
String must be enquotes, see our YAML style guide in the developer documentation.

```suggestion
      message: "Vehicle plugged in"
```